samples/javascript_nodejs/50.teams-messaging-extensions-search/bots/teamsMessagingExtensionsSearchBot.js

```diff
@@ -52,42 +52,42 @@ class TeamsMessagingExtensionsSearchBot extends TeamsActivityHandler {
      GetAdaptiveCard() {
 
         const preview = CardFactory.thumbnailCard(
-            'Adaptive Card',
-            'Please select to get the card'
+                'Adaptive Card',
+                'Please select to get the card'
         );
 
         const adaptive = CardFactory.adaptiveCard(AdaptiveCard);
         
-        const attachment = { ...adaptive,preview };
+        const attachment = { ...adaptive, preview };
         return {
             composeExtension: {
-            type: 'result',
-            attachmentLayout: 'list',
-            attachments: [attachment]
+                   type: 'result',
+                   attachmentLayout: 'list',
+                   attachments: [attachment]
             }
         };
     }
     
     GetConnectorCard() {    
         const preview = CardFactory.thumbnailCard(
-        'Connector Card',
-        'Please select to get the card'
+                'Connector Card',
+                'Please select to get the card'
         );
 
         const connector = CardFactory.o365ConnectorCard(ConnectorCard);
-        const attachment = {...connector,preview}
+        const attachment = {...connector, preview };
         return {
             composeExtension: {
-            type: 'result',
-            attachmentLayout: 'list',
-            attachments: [attachment]
+                   type: 'result',
+                   attachmentLayout: 'list',
+                   attachments: [attachment]
             }
         };
     }
    
     GetResultGrid() {
         const attachments = [];    
-        var files = fs.readdirSync(publicDir,function(err, result) {
+        let files = fs.readdirSync(publicDir,function(err, result) {
```

Comment:
`GetResultGrid()` could use a re-write to:

```js
    GetResultGrid() {
        const attachments = [];    
        const files = fs.readdirSync(publicDir, (err, result) => {
            if(err) {    
               console.error('error', err);
            }
        });

        files.forEach((file) => {
            const grid = CardFactory.thumbnailCard(
                '',
                [{ url: `${baseurl}/Images/${file}` }]
            );

            attachments.push(grid);
       });
       
        return {
            composeExtension: {
                   type: 'result',
                   attachmentLayout: 'grid',
                   attachments: attachments
                }
            };     
        }
```

Sorry, because it touches lines not touched by the PR, it won't let me use the GitHub Suggestion feature. This might copy/paste a little funny, so watch out for whitespace issues.